18F/identity-site
# LG-10330 Enable JSX and Turn PostOffSearch into JSX Component

## 🎫 Ticket
[LG-10330](https://cm-jira.usa.gov/browse/LG-10330)  Make PO Search's no results view configurable

This is the first step (more PR to come) to be able to pass in a component to AddressSearch

## 🛠 Summary of changes
- installed babel/preset-react v7.22.5 package as dev dependency
- added babel/preset-react to the presents list in babel.config.json
- used JSX in post_office_search.js

## 📜 Testing Plan

Provide a checklist of steps to confirm the changes.

1. Change show_po_search from false to true in _config.yml
2. Start sever
3.  Vist `http://localhost:4000/help/verify-your-identity/verify-your-identity-in-person/find-a-participating-post-office/`
4. Observe component on page with no errors in console (no changes to the components look/behavior)
5. Poke around other areas of the app to confirm everything else is working as expected

## 📸 Screenshots

![Screenshot 2023-08-31 at 1 57 42 PM](https://github.com/18F/identity-site/assets/125507397/6aee7764-e420-4121-84c6-fe84b82df9b0)
